spec/interactions/list_post_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe ListPosts do
+  describe 'interactor' do
+    context 'no posts' do
+      it {
```

Comment:
przy jednolinijkowych testach stosujemy `it { }`
a przy wielolinijkowych, jak tutaj 
```
it do
[...]
end
```

Reply:
http://www.betterspecs.org/ - tutaj jest zbiór wytycznych przyjętych w środowisku rails odnośnie rspec'a. Nie jest to bardzo istotna uwaga, chciałem raczej pokazać, że takie wytyczne są zdefiniowane.